hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/TokenServiceImpl.java

```diff
@@ -88,35 +90,48 @@ public Set<StateDefinition> statesToCreate() {
             public void migrate(@NonNull MigrationContext ctx) {
                 // TBD Verify this is correct. We need to preload all the special accounts
```

Comment:
Is this comment still valid?  I kind of thought this is what's going on at L108-L135 now.

Reply:
Yes, I just did a quick and dirty implementation, but a full one needs to be done still.